tests/test_codecs.py

```diff
@@ -299,6 +299,60 @@ def test_zipkin_codec_inject(self):
         assert carrier == {'span_id': 127, 'parent_id': None,
                            'trace_id': 256, 'traceflags': 1}
 
+    def test_zipkin_b3_codec_inject(self):
+        codec = B3Codec()
+
+        with self.assertRaises(InvalidCarrierException):
+            codec.inject(span_context=None, carrier=[])
+
+        ctx = SpanContext(trace_id=256, span_id=127, parent_id=None, flags=1)
+        span = Span(context=ctx, operation_name='x', tracer=None, start_time=1)
+        carrier = {}
+        codec.inject(span_context=span, carrier=carrier)
+        assert carrier == {'X-B3-SpanId': format(127, 'x').zfill(16),
+                           'X-B3-TraceId': format(256, 'x').zfill(16), 'X-B3-Flags': '1'}
```

Comment:
> `format(127, 'x').zfill(16)`

nit: this is a faulty way of writing tests, because it pretty much boils down to "to test my production function f(x) I am going to assert f(x) == f(x)". A better test is to compare the result with a value that is not produced by the same code as prod code, i.e. `'X-B3-SpanId': 0x000000000000007F`.